src/metax_api/tests/api/rest/base/views/contracts/contracts.py

```diff
@@ -123,6 +125,8 @@ def test_delete_contract(self):
             raise Exception('Deleted contract should not be deleted from the db')
 
         self.assertEqual(deleted_contract.removed, True, 'Deleted contract should be marked removed in the db')
+        self.assertTrue(deleted_contract.date_modified >= get_tz_aware_now_without_micros() - timedelta(seconds=30),
+            'date_modified should be updated')
```

Comment:
For delete tests, it may be simpler to simply test that the new value of date_modified
1) is the same as value of date_removed
2) and is newer than the old value of date_modified, instead of using timedelta.